spec/blueberry_rails_spec.rb

```diff
@@ -1,65 +1,33 @@
 require 'minitest/autorun'
-require 'fileutils'
+require './spec/support/blueberry_rails_helpers.rb'
 
-class BlueberryRailsTest < Minitest::Unit::TestCase
+class BlueberryRailsTest < Minitest::Test
+  include BlueberryRailsHelpers
 
   def setup
-    FileUtils.rm_rf 'test_project'
+    remove_project_directory
     cmd 'dropdb test_project_development'
     cmd 'dropdb test_project_test'
-
-    %w[RUBYOPT BUNDLE_PATH BUNDLE_BIN_PATH BUNDLE_GEMFILE].each do |key|
-      set_env key, nil
-    end
-  end
-
-  def teardown
-    restore_env
-  end
-
-  def test_rake_runs_cleanly
-    create_project 'test_project'
-    assert run_rake('test_project')
-  end
-
-  def test_rake_runs_cleanly_with_bootstrap_option
-    create_project 'test_project', '--bootstrap'
-    assert run_rake('test_project')
   end
 
-  protected
-
-  def create_project(project_name, options = nil)
-    bin = File.expand_path(File.join('..', 'bin', 'blueberry_rails'), File.dirname(__FILE__))
-    cmd "#{bin} #{project_name} #{options}"
-  end
-
-  def run_rake(project_name)
-    Dir.chdir(project_name) do
-      cmd 'bundle exec rake db:create'
-      cmd 'bundle exec rake db:migrate'
-      cmd 'bundle exec rake db:test:prepare'
-      cmd 'bundle exec rake'
+  def run
+    Bundler.with_clean_env do
+      super
     end
   end
 
-  def cmd(command)
-    system "#{command}"
-  end
+  def test_rake_runs_cleanly
+    create_project
 
-  def original_env
-    @original_env ||= {}
+    assert_exist_file 'config', 'initializers', 'simple_form.rb'
+    assert run_rake
   end
 
-  def set_env(key, value)
-    original_env[key] = ENV.delete(key)
-    ENV[key] = value
-  end
+  def test_rake_runs_with_bootstrap_option
+    create_project '--bootstrap'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.